src/ontology/nbo-edit.owl

```diff
@@ -9949,6 +9954,37 @@ AnnotationAssertion(oboInOwl:hasOBONamespace obo:NBO_0050002 "behavior_ontology"
 AnnotationAssertion(rdfs:label obo:NBO_0050002 "circling behavior")
 SubClassOf(obo:NBO_0050002 obo:NBO_0000013)
 
+# Class: obo:NBO_0070001 (migration)
+
+AnnotationAssertion(obo:IAO_0000115 obo:NBO_0070001 "Rhythmic behavior that involves cue-initiated, persistent, direct, focused movement between habitats.")
+AnnotationAssertion(terms:contributor obo:NBO_0070001 <https://orcid.org/0000-0002-4277-2151>)
+AnnotationAssertion(terms:date obo:NBO_0070001 "2023-04-22T14:17:43Z"^^xsd:dateTime)
+AnnotationAssertion(oboInOwl:hasRelatedSynonym obo:NBO_0070001 "Movement ecology")
+AnnotationAssertion(oboInOwl:source obo:NBO_0070001 "Dingle H. Migration: the biology of life on the move. Oxford University Press, USA; 2014.")
+AnnotationAssertion(rdfs:label obo:NBO_0070001 "migration")
+SubClassOf(obo:NBO_0070001 obo:NBO_0000008)
+
+# Class: obo:NBO_0070002 (diel vertical migration)
+
+AnnotationAssertion(obo:IAO_0000115 obo:NBO_0070002 "Migration that involves regular movement up and down in the water column.")
+AnnotationAssertion(dc:contributor obo:NBO_0070002 <http://orcid.org/0000-0002-2908-3327>)
+AnnotationAssertion(terms:contributor obo:NBO_0070002 <https://orcid.org/0000-0002-4277-2151>)
+AnnotationAssertion(terms:date obo:NBO_0070002 "2023-04-22T14:41:53Z"^^xsd:dateTime)
+AnnotationAssertion(oboInOwl:source obo:NBO_0070002 "Bandara K, Varpe Ø, Wijewardene L, Tverberg V, Eiane K. Two hundred years of zooplankton vertical migration research. Biological Reviews. 2021 Aug;96(4):1547-89.")
+AnnotationAssertion(rdfs:label obo:NBO_0070002 "diel vertical migration")
+SubClassOf(obo:NBO_0070002 obo:NBO_0070001)
+
+# Class: obo:NBO_0070003 (geographical migration)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1146/annurev-ecolsys-012021-031035") obo:IAO_0000115 obo:NBO_0070003 "Migration that involves regular movement laterally between distant ranges in seasonal cycles.")
+AnnotationAssertion(dc:contributor obo:NBO_0070003 <http://orcid.org/0000-0002-2908-3327>)
+AnnotationAssertion(terms:contributor obo:NBO_0070003 <https://orcid.org/0000-0002-4277-2151>)
```

Comment:
You sometimes use `terms:contributor` and other times `dc:contributor` - you should only ever use the former.